README.md

```diff
@@ -20,6 +20,8 @@ Add the configuration to the MCP server configs (e.g. in Cline when using VSCode
         "-Dtransport.mode=stdio",
         "-Dspring.main.web-application-type=none",
         "-Dlogging.file.name=/spring-io-api-mcp.log",
+         "-Dtransport.mode=stdio",
+        "-Dspring.main.web-application-type=none",
```

Comment:
duplicated, see lines above

Reply:
Done. Removed redundant args from README example configuration for MCP client.